charactersheet/charactersheet/models/common/armor.js

```diff
@@ -25,7 +25,12 @@ function Armor() {
     self.armorStealthOptions = ko.observableArray(Fixtures.armor.armorStealthOptions);
 
     self.acLabel = ko.pureComputed(function() {
-        return 'AC ' + self.armorClass();
+        if (self.armorClass().replace(/^\s+|\s+$/g, '')) {
```

Comment:
I actually think we should not trim for one reason; we don't do it anywhere else. There are various places where we check for a value, and if it's a blank space, we count it as a value. My opinion is we either remove the trimming or add it everywhere.